### PR TITLE
contributing: update to reference single line braces

### PR DESCRIPTION
**Pull request by maryamtahhan**
Signed-off-by: Maryam Tahhan <mtahhan@redhat.com>

### CODE REVIEW

**Comment by maryamtahhan** [comment]
I tried looking into clang-tidy as an option to fix this - but didn't find anything straight forward bar the clang-format option called InsertBraces, but it doesn't arrive till clang-format version 15... 

https://github.com/CloudNativeDataPlane/cndp/issues/246

**Review by jeffreybshaw — APPROVED** [review]
This content is duplicated on https://github.com/CloudNativeDataPlane/cloudnativedataplane.github.io/blob/main/dev.md, do you want to update it there, too? We can create a reminder issue in that repo (or this one) so we don't forget.

**Comment by maryamtahhan** [comment]
> This content is duplicated on https://github.com/CloudNativeDataPlane/cloudnativedataplane.github.io/blob/main/dev.md, do you want to update it there, too? We can create a reminder issue in that repo (or this one) so we don't forget.

I will push an update there too :)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ This document outlines how to contribute code to the CNDP project.
 The CNDP code can be cloned from the repository on GitHub:
 
 ``` bash
-git clone https://github.com/CloudNativeDataPlane/cndp.git
+$ git clone https://github.com/CloudNativeDataPlane/cndp.git
```
[the remark this answers is not in the document]
Comment (maryamtahhan) [reply CONTRIBUTING.md:10]: I will fix them up so the linter doesn't have a hernia :)

**Review by KeithWiles — APPROVED** [review]
LGTM, whatever you want to do for the other issue

**Comment by maryamtahhan** [comment]
> https://github.com/CloudNativeDataPlane/cloudnativedataplane.github.io/blob/main/dev.md

I added a PR https://github.com/CloudNativeDataPlane/cloudnativedataplane.github.io/pull/2

**Review by KeithWiles — APPROVED** [review]

**Pull request merged**